<launch>

	<node pkg="sensor_sub" type="compass_pub" name="compass_pub">
		<param name="freq"   value="2"/>
		<param name="serial_port"   value="/dev/ttyUSB0"/>
	</node>

    <node pkg="centre_control" type="compass_pid_test" name="compass_pid_test" output="screen">
		<rosparam param="ang_kp_kd_ki_1" >[10,8,1]</rosparam>
		<rosparam param="ang_kp_kd_ki_2" >[10,8,0]</rosparam>
		<param name="l_acc_factor"	type = "double"	value="-5.0"/>
		<param name="r_acc_factor"	type = "double"	value="5.0"/>
		<param name="ang_dt"	type = "double"	value="0.5"/>
		<param name="ang_pwm_max"	type = "int"	value="300"/>
		<param name="serial_port"	value="/dev/ttyUSB1"/>
	</node>

</launch>


